lisa/node.py

```diff
@@ -267,6 +267,7 @@ def execute(
         no_debug_log: bool = False,
         cwd: Optional[PurePath] = None,
         timeout: int = 600,
+        encoding: str = "utf-8",
```

Comment:
Because the encoding of node won't be changed, so it can be set when node is created.

Reply:
OK, I can store this state in the node.